2-mandatory/3-shopping-list.js

```diff
@@ -20,8 +20,15 @@ let pantry = {
 
 function createShoppingList(recipe) {
   // write code here
+  let allContent = pantry.fridgeContents.concat(pantry.cupboardContents);
+  let missingIngredients = recipe.ingredients
+  .filter(ingredient => allContent.indexOf(ingredient) === -1);
+  return {name: recipe.name ,
```

Comment:
well done Hengameh :). you can also do this with includes  instead of indexOf

Reply:
sure